var/spack/repos/builtin/packages/bedtools2/package.py

```diff
@@ -29,5 +29,7 @@ class Bedtools2(Package):
     depends_on("xz", when="@2.29:")
     depends_on("python", type="build")
 
+    patch("bedtools-gcc13.patch", level=1, when="@2.27:%gcc@13")
```

Comment:
```suggestion
    patch("bedtools-gcc13.patch", level=1, when="@2.27:%gcc@13:")
```

@A-N-Other it might be worth applying this patch to all gcc versions 13 and later in the case that someone might want to build v2.27 with gcc 14 one day.